### PR TITLE
Update chart README with section on Airflow configuration

### PR DESCRIPTION
**Pull request by lewismc**
Being new to the Helm Chart, I asked a [question](https://lists.apache.org/thread.html/r157160eab6e8ae714ab0dfb0a22125059fe1bffada871fb5e24f3bfa%40%3Cusers.airflow.apache.org%3E) on user@airflow.a.o. I finally discovered that the Airflow configuration parameters can be entered in [values.yaml](https://github.com/apache/airflow/blob/master/chart/values.yaml).

This simple PR should hopefully help anyone else in a similar situation to me.

### CODE REVIEW

File: chart/README.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ cluster using the [Helm](https://helm.sh) package manager.
 - Helm 2.11+ or Helm 3.0+
 - PV provisioner support in the underlying infrastructure
 
+## Configuring Airflow
+
+All Airflow configuation parameters (equivalent of `airflow.cfg`) are stored in [values.yaml](https://github.com/apache/airflow/blob/master/chart/values.yaml). You may wish to consult these prior to Chart installation.
+
```
Comment (kaxil) [inline chart/README.md]: The description is not clear enough, it also does not tell which key needs to be set, all configs need to go under `config` key:

https://github.com/apache/airflow/blob/9c75ea3c14b71d2f96d997aeef68c764c7d2984c/chart/values.yaml#L643

Would be better to give an example too

Comment (lewismc) [reply chart/README.md]: Thank you for the suggestion @kaxil I appreciate that. I've augmented the PR to accommodate.

File: chart/README.md [hunk 2]
```diff
@@ -32,6 +32,47 @@ cluster using the [Helm](https://helm.sh) package manager.
 - Helm 2.11+ or Helm 3.0+
 - PV provisioner support in the underlying infrastructure
 
+## Configuring Airflow
+
+All Airflow configuation parameters (equivalent of `airflow.cfg`) are stored in [values.yaml](https://github.com/apache/airflow/blob/master/chart/values.yaml) under the `config` key (around line 650 of `values.yaml`). The following code demonstrates how one would deny webserver users from viewing the config from within the webserver application. See the bottom line of the example...
```
Comment (kaxil) [inline chart/README.md]:
```suggestion
All Airflow configuration parameters (equivalent of `airflow.cfg`) are stored in [values.yaml](https://github.com/apache/airflow/blob/master/chart/values.yaml) under the `config` key . The following code demonstrates how one would deny webserver users from viewing the config from within the webserver application. See the bottom line of the example:
```

File: chart/README.md [hunk 3]
```diff
@@ -32,6 +32,47 @@ cluster using the [Helm](https://helm.sh) package manager.
...
+
+```
```
Comment (kaxil) [inline chart/README.md]:
```suggestion
```yaml
```

File: chart/README.md [hunk 4]
```diff
@@ -32,6 +32,47 @@ cluster using the [Helm](https://helm.sh) package manager.
...
+    colored_console_log: 'False'
+    logging_level: DEBUG
+  metrics:
+    statsd_on: '{{ ternary "True" "False" .Values.statsd.enabled }}'
+    statsd_port: 9125
+    statsd_prefix: airflow
+    statsd_host: '{{ printf "%s-statsd" .Release.Name }}'
+  webserver:
+    enable_proxy_fix: 'True'
+    expose_config: 'False'    <<<<<<<<<< BY DEFAULT THIS IS 'True' BUT WE CHANGE IT TO 'False' PRIOR TO INSTALLING THE CHART
```
Comment (kaxil) [inline chart/README.md]:
```suggestion
    expose_config: 'False'   # <<<<<<<<<< BY DEFAULT THIS IS 'True' BUT WE CHANGE IT TO 'False' PRIOR TO INSTALLING THE CHART
```

File: chart/README.md [hunk 5]
```diff
@@ -32,6 +32,47 @@ cluster using the [Helm](https://helm.sh) package manager.
...
+```
+
+Generally speaking, it is useful to familiarize ones self with the Airflow configuration prior to installing and deploying the service.
```
Comment (kaxil) [inline chart/README.md]:
```suggestion
Generally speaking, it is useful to familiarize oneself with the Airflow configuration prior to installing and deploying the service.
```

**Review by kaxil — APPROVED** [review]

**Pull request merged**